### PR TITLE
Add attribution for script icon

### PR DESCRIPTION
**Pull request by Technical-Coder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,5 @@ The documentation for Mathscript can be found [here](https://eyx092.github.io/ma
 ## Questions
 
 Ask questions in the 'Issues' tab.
+
+*The [script icon](https://thenounproject.com/term/code-script/691897/) in the Mathscript logo was designed by [Shmidt Sergey](https://thenounproject.com/monstercritic/).*
```
Comment (eyx092) [inline README.md]:
```suggestion
<sub><sup>*The [script icon](https://thenounproject.com/term/code-script/691897/) in the Mathscript logo was designed by [Shmidt Sergey](https://thenounproject.com/monstercritic/).*</sup></sub>
```

**Review by eyx092 — CHANGES_REQUESTED** [review]
I just wanted to make the text smaller if that's good with you.

**Comment by Technical-Coder** [comment]
Yeah that is fine.

**Review by eyx092 — APPROVED** [review]
Approved 👍

**Pull request merged**